lib/import-export-visitor.js

```diff
@@ -70,13 +70,14 @@ module.exports = class ImportExportVisitor extends Visitor {
       }
 
       bodyInfo.body = null;
-      bodyInfo.parent = null;
       bodyInfo.hoistedExportsMap = null;
       bodyInfo.hoistedExportsString = null;
       bodyInfo.hoistedImportsString = null;
       bodyInfo.insertCharIndex = null;
       bodyInfo.insertNodeIndex = null;
       bodyInfo.needToAddUseStrictDirective = null;
+      bodyInfo.parent._bodyInfoByName = null;
```

Comment:
This is the last bit of pseudo private stuff, `parent._bodyInfoByName`. I noticed while we cleared body info on the visitor we didn't clear it on the ast node. So I went ahead and did that too.